src/lib/s3-helper.js

```diff
@@ -70,7 +71,7 @@ class S3Helper {
    * @returns {Promise<Object>} - Map of file hashes
    */
   getRemoteHashesMap() {
-    return this._s3Client.getObject({ Key: this._mapFileName })
+    return this._s3Client.getObject({ Key: this._keyPrefix + this._mapFileName })
```

Comment:
I believe we can prefix `_mapFileName` on the initialization, what do you think?